modules/nf-core/proteinortho/main.nf

```diff
@@ -0,0 +1,54 @@
+process PROTEINORTHO {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "bioconda::proteinortho=6.3.0"
```

Comment:
We've recently swapped to putting the conda environment into a separate file.
I'm on my phone ATM so linking is hard, but if you take a look at any of the modules that are currently in you should see the new structure.

Reply:
implemented